Tests/MapboxMapsTests/Foundation/Extensions/UIWindow+ParentSceneTests.swift

```diff
@@ -0,0 +1,20 @@
+import XCTest
+import CarPlay
+@testable import MapboxMaps
+
+final class UIWindowParentSizeTests: XCTestCase {
+
+    @available(iOS 13.0, *)
+    func testCarPlayWindowReturnsCorrectParentScene() {
+        let window = CPWindow()
+
+        XCTAssertEqual(window.parentScene, window.templateApplicationScene)
+    }
+
+    @available(iOS 13.0, *)
```

Comment:
Please make sure the tests pass on OS versions < 13

Reply:
I vaguely recall this not working in tests and needing to do runtime checks in the test implementation.

Reply:
But test it to make sure I'm not misremembering.

Reply:
Good point, indeed it just crashes when running e.g. on iOS 12 with these annotations. Converted them to runtime checks.